sdk/servicebus/service-bus/test/stress/runContainer.js

```diff
@@ -29,16 +29,22 @@ const options = minimist(process.argv, {
   string: ["testToRun"]
 });
 
-if (!options.testToRun || !fs.existsSync(path.join("dist", options.testToRun))) {
-  console.error("Usage: runcontainer.js --testToRun <name of file in `dist`>");
+if (!options.testToRun) {
+  console.error("Usage: runcontainer.js --testToRun <name of a stress test file`>");
   process.exit(1);
 }
 
+console.log(`Running \`npm run build\``);
+spawn(`npm run build`);
+
 const { name: testFileNameWithoutExtension } = path.parse(options.testToRun);
 const containerInstanceName = `${resourceGroup}-sb-stressperf-${testFileNameWithoutExtension.toLowerCase()}`;
 
-console.log(`Running \`npm run build\``);
-spawn(`npm run build`);
+const expectedJsFile = path.join(`dist`, `${testFileNameWithoutExtension}.js`);
+if (!fs.existsSync(expectedJsFile)) {
+  console.error(`No file named ${expectedJsFile}. Need a valid JS file when creating your container group.`);
+  process.exit(1);
+}
 
```

Comment:
For the `az` commands below, I had to pass the subscription-id too. https://github.com/Azure/azure-sdk-for-js/commit/47da2e403213d5dd79c56a7d79122e0a69c44229

Reply:
Ah, yes you're right. I like your change, I'll add it in.

In the future (if you ever want to avoid having to pass in the subscription ID, especially when you have one primary subscription) you can do:

`az accounts set --subscription <subscription-id>` which makes it implicit to commands from that point. (it can still be overridden on a command by command basis, however).

Reply:
Fixed!